compiler/AST/astutil.cpp

```diff
@@ -655,36 +679,52 @@ bool isTypeExpr(Expr* expr)
   return false;
 }
 
+static void pruneVisit(TypeSymbol* ts, Vec<FnSymbol*>& fns, Vec<TypeSymbol*>& types);
```

Comment:
There seem to me to be 3 variations of pruneVisit().  Two are named pruneVisit and one is named printVisitFn().
Inconsistentcy or deliberate?